src/library/dlganalysis.h

```diff
@@ -34,6 +34,12 @@ class DlgAnalysis : public QWidget, public Ui::DlgAnalysis, public virtual Libra
     inline const QString currentSearch() {
         return m_pAnalysisLibraryTableModel->currentSearch();
     }
+    void saveCurrentViewState() override {
+        m_pAnalysisLibraryTableView->saveCurrentViewState();
```

Comment:
Move into .cpp file and revert the inclusion of wanalysislibrarytableview.h in the header file. Please do this for all implementations of these functions.